IbisLib/scenemanager.cpp

```diff
@@ -1659,20 +1659,14 @@ void SceneManager::ObjectWriter( Serializer * ser )
             QFileInfo fi(obj->GetDataFileName());
             dataFileName.append(fi.completeSuffix());
             newPath.append(dataFileName);
-            if( obj->IsA( "PolyDataObject" )) // from now on, we always save points in scene.xml
+            if( !obj->IsA( "PointsObject" )) // from now on, we always save points in scene.xml
```

Comment:
Should the if conditions be inverted instead? to facilitate readibility and future object saving:
if (obj->IsA( "PointsObject" ))
{
   // do specific processing for PointsObject
}
else
{
    // do generic processing for file copy
}